solarforecastarbiter/metrics/tests/test_preprocessing.py

```diff
@@ -421,6 +426,7 @@ def test_align_prob_constant_value(
                 [1.], index=pd.DatetimeIndex(["20200301T03Z"], freq='1h')),
             (('ISNAN', 1), ('NIGHTTIME', 2), ('USER FLAGGED', 2),
              ('TOTAL DISCARD BEFORE RESAMPLE', 3),
+             ('Outage', 0),
```

Comment:
Not sure how I feel about this being the only non-caps thing. Any reason for that?